mud-contracts/world-v2/.env

```diff
@@ -9,14 +9,14 @@ DEBUG=mud:*
 #
 # Anvil default private key:
 PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
-RPC_URL=http://localhost:8545
+RPC_URL=http://127.0.0.1:8545
 WORLD_ADDRESS=
 
 
 ERC20_TOKEN_NAME=EVE TOKEN
 ERC20_TOKEN_SYMBOL=EVE
 ERC20_INITIAL_SUPPLY=10000000000
-EVE_TOKEN_NAMESPACE=eveERC20
+EVE_TOKEN_NAMESPACE=test
```

Comment:
If this is the core set of contracts for the world and not something primarily targeted at builder development, do we need to rename the namespace to `test` instead of keeping the existing namespace?